### PR TITLE
Add checkdeps command

### PR DESCRIPTION
**Pull request by sbadithe**
- Should fix broken runs of `latest_dependency_checker.yaml` by adding missing Makefile command

### CODE REVIEW

File: meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "featuretools_sql" %}
```
Comment (gsheni) [inline meta.yaml]: This file should not be here.

File: Makefile [hunk 2]
```diff
@@ -46,6 +46,11 @@ upgradebuild:
 upgradesetuptools:
 	python -m pip install --upgrade setuptools
 
+.PHONY: checkdeps
+checkdeps:
+	$(eval allow_list='numpy|pandas|featuretools|psycopg2|sqlalchemy|PyMySQL|snowflake-sqlalchemy[pandas]')
+	pip freeze | grep -v "github.com" | grep -E $(allow_list) > $(OUTPUT_PATH)
```
Comment (gsheni) [inline Makefile:52]:
```suggestion
	pip freeze | grep -v "alteryx/featuretools_sql" | grep -E $(allow_list) > $(OUTPUT_PATH)
```

**Review by gsheni — COMMENTED** [review]
looks good except for the 1 comment

File: Makefile (hunk 2 above)
Comment (sbadithe) [reply Makefile]: changed to `alteryx/featuretools*` to handle both featuretools and featuretools_sql lines

Comment (gsheni) [reply Makefile]: Wait this isn't meant to be a catch all. It should be repo specific.

**Review by gsheni — APPROVED** [review]

**Pull request merged**